mqlight/src/main/java/com/ibm/mqlight/api/NonBlockingClientAdapter.java

```diff
@@ -18,27 +18,41 @@
  */
 package com.ibm.mqlight.api;
 
+import com.ibm.mqlight.api.impl.NonBlockingClientImpl;
+import com.ibm.mqlight.api.logging.Logger;
+import com.ibm.mqlight.api.logging.LoggerFactory;
+
 /**
- * An abstract adapter class for receiving client events. The methods in this class are empty.  
+ * An abstract adapter class for receiving client events. The methods in this class are empty.
  * This class exists as convenience for creating client listener objects.  Extend this class to
- * create a <code>NonBlockingClientListener</code> object and override the methods for the events of interest. 
+ * create a <code>NonBlockingClientListener</code> object and override the methods for the events of interest.
  * (If you implement the <code>NonBlockingClientListener</code> interface, you have to define all of the methods in it.
  * This abstract class defines null methods for them all, so you can only have to define methods for
  * events you care about.)
  */
 public abstract class NonBlockingClientAdapter<T> implements NonBlockingClientListener<T> {
 
+    private static final Logger logger = LoggerFactory.getLogger(NonBlockingClientImpl.class);
```

Comment:
@prestona did you intentionally mean to use NonBlockingClientImpl for the logger name rather than NonBlockingClientAdapter?